src/plugins/kibana/index.js

```diff
@@ -22,7 +22,8 @@ module.exports = function (kibana) {
           'spyModes',
           'fieldFormats',
           'navbarExtensions',
-          'settingsSections'
+          'settingsSections',
+          'docTableDetailViews'
```

Comment:
I created kbn_doc_views plugin and moved them there.  The registry is now named docViews.